tf_patches/profiler_trace.diff

```diff
@@ -0,0 +1,611 @@
+diff --git a/tensorflow/python/profiler/internal/BUILD b/tensorflow/python/profiler/internal/BUILD
```

Comment:
lol what's this diff about?

Reply:
Added comment to this file and PR description.